commons_compress_7z/src/main/java/com/amaze/filemanager/filesystem/compressed/sevenz/SevenZFile.java

```diff
@@ -51,6 +52,7 @@
 import org.apache.commons.compress.utils.ByteUtils;
 import org.apache.commons.compress.utils.CRC32VerifyingInputStream;
 import org.apache.commons.compress.utils.InputStreamStatistics;
+import org.jetbrains.annotations.Nullable;
```

Comment:
Prefer not using `@Nullable` that is not from official Android SDK. Better pull it off at all.